### DISCUSSION TITLE
Accessing validation message from alpinejs

### DISCUSSION THREAD

**Discussion by toriqsetiawan** [livewire/livewire#5188] [1. Help]
Hi All,
could you please help me how to access error validation message from alpinejs?

```
this.$wire.fooFunction()
    .then( result => {  
        ....
    })
    .catch( error => {
        ....
    })
```
or maybe there is another approach?
thanks

**Comment by joshhanley** [comment]
@toriqsetiawan the error you would be catching is if there was an error with the ajax request, not if there was validation errors (as Livewire will still have a successful result).

If you need to display validation errors in the front end, you could dispatch a browser event with the errors as the payload, and then listen for it in the front end and handle it however you want.

Hope this helps!
